gui-apps/foot/foot-1.20.2-r1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson systemd verify-sig xdg
+
+DESCRIPTION="Fast, lightweight and minimalistic Wayland terminal emulator"
+HOMEPAGE="https://codeberg.org/dnkl/foot"
+SRC_URI="
+	https://codeberg.org/dnkl/foot/releases/download/${PV}/${P}.tar.gz
+	verify-sig? ( https://codeberg.org/dnkl/foot/releases/download/${PV}/${P}.tar.gz.sig )
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv"
+IUSE="+grapheme-clustering test utempter verify-sig"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
+	dev-libs/wayland
+	media-libs/fcft
+	media-libs/fontconfig
+	x11-libs/libxkbcommon
+	x11-libs/pixman
+	grapheme-clustering? (
+		dev-libs/libutf8proc:=[-cjk]
+		media-libs/fcft[harfbuzz]
+	)
+"
+DEPEND="
+	${COMMON_DEPEND}
+	>=dev-libs/tllist-1.1.0
+	>=dev-libs/wayland-protocols-1.32
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	|| (
+		~gui-apps/foot-terminfo-${PV}
+		>=sys-libs/ncurses-6.3[-minimal]
```

Comment:
WRT this change (and I won't consider it a blocker because.. well.. it did nothing), this won't have an effect, because most people don't use `ncurses[-minimal]` (and, indeed, I would not encourage `minimal`).

if you find a way to teach ncurses to prefer `foot-terminfo` over its own copy of the `foot` terminfo, that'd be nice, but until such a thing is found, I intentionally preferred ncurses' version of foot-terminfo to `foot-terminfo` as it means the user does not need to enable `minimal`.  doing so would mean that the user loses various terminfos, which makes it annoying to ssh for instance